pegasus/sites.v3/hourofcode.com/public/css/front-page.css

```diff
@@ -53,3 +54,44 @@
 .category-link:hover {
   text-decoration: none;
 }
+
+#textbacking {
+  margin-top: 80px;
+  color: white;
+  border-radius: 5px;
+  padding: 10px;
+  text-shadow: 0px 0px 4px rgba(31, 31, 31, 1.0);
+}
+
+.watch-video {
+  color:white;
+  text-shadow: 0px 0px 4px rgba(31, 31, 31, 1);
+  margin-top: 0px;
+  margin-bottom:10px;
+  padding:20px
+}
+
+#leftbutton {
+  position:absolute;
+  left:0;
+  top:0;
+  text-align:center;
```

Comment:
we should be consistent on whether there's a space after the colon or not.  i think there should be.